controller/src/beerocks/cli/beerocks_cli_bml.cpp

```diff
@@ -1783,25 +1572,19 @@ int cli_bml::get_slave_restricted_channels(const std::string &hostap_mac)
 
 int cli_bml::wfca_controller(const std::string &cmd)
 {
-    int ret = bml_wfca_controller(ctx, cmd.c_str(), print_buffer, sizeof(print_buffer));
-    std::string ret_str;
-    if (!wfca_dummy_ret_val_gen(cmd, ret_str)) {
-        ret = BML_RET_INVALID_ARGS;
+    int ret = bml_wfa_ca_controller(ctx, cmd.c_str(), cmd.size() + 1, wfa_ca_reply_to_console_cb);
+
+    // print error message only on error
+    if (ret != BML_RET_OK) {
+        printBmlReturnVals("wfca_controller", ret);
     }
-    print_bml_wfca_ret_val("wfca_controller", ret, ret_str);
 
     return 0;
 }
 
 int cli_bml::wfca_agent(const std::string &cmd)
 {
-    int ret = bml_wfca_controller(ctx, cmd.c_str(), print_buffer, sizeof(print_buffer));
-    std::string ret_str;
-    if (!wfca_dummy_ret_val_gen(cmd, ret_str)) {
-        ret = BML_RET_INVALID_ARGS;
-    }
-    print_bml_wfca_ret_val("wfca_agent", ret, ret_str);
-
+    // TODO
```

Comment:
I don't know currently what has to be done. This will be part of the agent certification.
It is irrelevant now, and I don't think it is a good idea to put effort into it now.